<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KaXmitCtlMainWindow</class>
 <widget class="QMainWindow" name="KaXmitCtlMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ka Transmitter Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>4</number>
    </property>
    <property name="margin">
     <number>4</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QGroupBox" name="statusBox">
        <property name="title">
         <string>General Status</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="margin">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="runupLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>HVPS Runup</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="standbyLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Standby</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="warmupLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Warmup</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cooldownLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Cooldown</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="hvpsOnLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>HVPS On</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="remoteEnabledLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remote Enabled</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="hvpsVoltageTitle">
              <property name="text">
               <string>HVPS Voltage </string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="hvpsVoltageValue">
              <property name="text">
               <string>00.0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="hvpsVoltateUnits">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>kV</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="magCurrentTitle">
              <property name="text">
               <string>Magnetron Avg. Current </string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="magCurrentValue">
              <property name="text">
               <string>00.0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="magCurrentUnits">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>mA</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="hvpsCurrentTitle">
              <property name="text">
               <string>HVPS Current</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="hvpsCurrentValue">
              <property name="text">
               <string>00.0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="hvpsCurrentUnits">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>mA</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="temperatureValue">
              <property name="text">
               <string>00</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="temperatureUnits">
              <property name="text">
               <string>C</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="temperatureTitle">
              <property name="text">
               <string>Temperature</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="autoResetTitle">
              <property name="text">
               <string>Auto Pulse Fault Resets</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="autoResetValue">
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="faultStatusBox">
        <property name="title">
         <string>Fault Status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>4</number>
         </property>
         <property name="horizontalSpacing">
          <number>-1</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="blowerFaultIcon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="blowerFaultLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Blower</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="interlockFaultIcon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="interlockFaultLabel">
           <property name="text">
            <string>Safety Interlock</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="revPowerFaultIcon">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="revPowerFaultLabel">
           <property name="text">
            <string>Reverse Power</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="pulseInputFaultIcon">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="pulseInputFaultLabel">
           <property name="text">
            <string>Pulse Input</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="wgPresFaultIcon">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="wgPresFaultLabel">
           <property name="text">
            <string>Waveguide Pressure</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="magCurrFaultIcon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="magCurrFaultLabel">
           <property name="text">
            <string>Magnetron Current</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="hvpsCurrFaultIcon">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="hvpsCurrFaultLabel">
           <property name="text">
            <string>HVPS Current</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="hvpsUnderVFaultIcon">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="hvpsUnderVFaultLabel">
           <property name="text">
            <string>HVPS Under Voltage</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="hvpsOverVFaultIcon">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="hvpsOverVFaultLabel">
           <property name="text">
            <string>HVPS Over Voltage</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="logArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::WidgetWidth</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="unitOnLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="KaXmitCtlMainWindow.qrc">:/greenLED_off.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="powerButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="KaXmitCtlMainWindow.qrc">
          <normaloff>:/powerbutton.png</normaloff>:/powerbutton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="faultResetButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fault Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="standbyButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Standby</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="operateButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Operate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="KaXmitCtlMainWindow.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>powerClicked()</slot>
  <slot>faultResetClicked()</slot>
  <slot>standbyClicked()</slot>
  <slot>operateClicked()</slot>
 </slots>
</ui>
